axelor-stock/src/main/resources/views/Product.xml

```diff
@@ -140,8 +140,123 @@
+      </replace>
+    </extend>
   </form>
 
+  <action-group name="action-stock-product-group-compute-volume-masses-density">
+    <action name="action-product-record-calculate-volume-length"/>
+    <action name="action-stock-product-record-compute-metric-mass"/>
+    <action name="action-stock-product-record-compute-net-mass"/>
+    <action name="action-stock-product-record-compute-density"/>
+  </action-group>
+
+  <action-group id="stock-action-group-product-onload"
+    name="action-group-product-onload">
+    <action name="action-product-attrs-scale-and-precision"/>
+    <action name="action-product-attrs-scale-and-precision-production"/>
+    <action name="action-product-attrs-scale-and-precision-stock"/>
+    <action name="action-product-attrs-title-unit"/>
+    <action name="action-product-attrs-in-ati"/>
+    <action name="action-product-attrs-change-title-based-on-inAti"/>
+    <action name="action-product-set-stock"/>
+    <action name="action-product-set-total-variants"/>
+    <action if="__context__?._fromSale" name="action-product-attrs-readonly-sellable"/>
+    <action name="action-product-attrs-set-required-productCategory"/>
+    <action name="action-product-attrs-enable-product-company-list"/>
+    <action name="action-product-attrs-set-cost-type-select"/>
+    <action name="action-product-attrs-set-required-procurement-method-select"/>
+    <action name="action-stock-product-attrs-set-scale-metric-mass-density"/>
+  </action-group>
+
+  <action-group id="stock-action-group-base-product-onnew"
+    name="action-group-base-product-onnew">
+    <action name="action-product-attrs-scale-and-precision"/>
+    <action name="action-product-attrs-scale-and-precision-production"/>
+    <action name="action-product-attrs-scale-and-precision-stock"/>
+    <action name="action-product-group-default-new"/>
+    <action name="action-product-record-sale-supply-select"/>
+    <action name="action-product-record-in-ati"/>
+    <action name="action-product-attrs-in-ati"/>
+    <action name="action-product-record-default-production"/>
+    <action name="action-product-record-sellable"/>
+    <action if="__context__?._fromSale" name="action-product-attrs-readonly-sellable"/>
+    <action if="__context__?._fromPurchase" name="action-product-attrs-readonly-purchasable"/>
+    <action name="action-product-record-set-barcode-type-config"/>
+    <action name="action-product-attrs-title-unit"/>
+    <action name="action-product-attrs-set-required-productCategory"/>
+    <action name="action-product-attrs-enable-product-company-list"/>
+    <action name="action-product-record-service-stock-managed"/>
+    <action name="action-product-attrs-set-cost-type-select"/>
+    <action name="action-product-attrs-set-required-procurement-method-select"/>
+    <action name="action-stock-product-attrs-set-scale-metric-mass-density"/>
+  </action-group>
```

Comment:
I think we should call a new action that calls the `action-group-base-product-onnew`, else this can become quickly difficult to maintain... same issue for onload